SPECS/mariner-repos/mariner-repos.spec

```diff
@@ -1,12 +1,12 @@
 Summary:        CBL-Mariner repo files, gpg keys
 Name:           mariner-repos
-Version:        1.0
-Release:        13%{?dist}
+Version:        2.0
```

Comment:
Are we going to have a separate repository for 2.0? If so, we should also update the .repo files themselves.

Reply:
Nevermind, I forgot the .repo files are parametrized.